packages/world/src/IWorldEvents.sol

```diff
@@ -0,0 +1,18 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.8.24;
+
+import { IWorldErrors } from "./IWorldErrors.sol";
+import { IModule } from "./IModule.sol";
+import { ResourceId } from "./WorldResourceId.sol";
+
+/**
+ * @title IWorldEvents
+ * @author MUD (https://mud.dev) by Lattice (https://lattice.xyz)
```

Comment:
should we have a dev comment like with errors?

Reply:
just realised that!

Reply:
added one :)